lib/parser.js

```diff
@@ -118,10 +111,17 @@ function text (scanner) {
 }
 
 /*
- * <scope>        ::= 1*<any UTF8-octets except newline or parens>
+ * "(" <scope> ")"        ::= 1*<any UTF8-octets except newline or parens>
  */
 function scope (scanner) {
+  if (scanner.peek() !== '(') {
+    return scanner.abort(scanner.enter('scope', ''))
```

Comment:
It's a bit awkward, but it works. Mainly because we throw away the `(` and `)` characters, so we need to initialize the node _after_ we consumed the first token.